release-notes/swan-lake-alpha3-release-note.md

```diff
@@ -31,20 +18,26 @@ If you are already using Ballerina, you can directly update your distribution to
 If you are using an **Update Tool version below 0.8.14**, execute the `ballerina update` command to update it. Next, execute the command below to update to Swan Lake Alpha3.
 
 > `bal dist pull slalpha3`
+<br><br>
 
-#### For New Users
+### Installing Ballerina
```

Comment:
```suggestion
### Installing Ballerina
```
Fixed in https://github.com/ballerina-platform/ballerina-dev-website/pull/2050.

Reply:
@keizer619 - Please review the new changes in https://github.com/ballerina-platform/ballerina-dev-website/pull/2050.